examples/apfl_example/client.py

```diff
@@ -15,6 +15,7 @@
 from fl4health.model_bases.apfl_base import ApflModule
 from fl4health.utils.load_data import load_mnist_data
 from fl4health.utils.metrics import Accuracy
+from fl4health.utils.random import set_all_random_seeds
```

Comment:
I know this is "out-of-scope" for this PR, but do you mind updating the typing annotation on `set_all_random_seeds`? Right now the seed argument is annotated as an `int`, but we really mean `Optional[int] `